Scripts/Options.gd

```diff
@@ -0,0 +1,47 @@
+extends Control
+
+enum eVideo { ONHOVER, FINISHED}
+enum eAudio { ONHOVER, FINISHED}
+enum eControls { ONHOVER, INISHED}
```

Comment:
FINISH is misspelled, also do you need these anymore since you have them in the button script?